### DISCUSSION TITLE
Is Mockoon compatible with cloud functions and serverless environments? I am currenlty working on Azure Functions

### DISCUSSION THREAD

**Discussion by dharma412** [mockoon/mockoon#1451] [General]
I am curretnly working on Azure Functions on trigger based. does mockoon compatiable with azure clode functions.

**Comment by 255kb** [comment]
Mockoon offers a [serverless package](https://github.com/mockoon/mockoon/blob/main/packages/serverless/README.md#request-listener-express-application) that is proven to work with Google Cloud Functions (incl. Firebase), AWS Lambda, Netlify. I guess it should work with Azure too. 
Let us know if there is any issue.

**Reply by dharma412** [reply]
I could not find any sample code to create function which run mockoon. Can you help me with any sample code. Currently we are creating azure functions based on java.

**Reply by 255kb** [reply]
Creating Express.js-based apps in Azure Functions appears to be more challenging compared to other providers. While there are no current plans to add official support for Azure, we welcome any assistance from the community.

Additionally, we offer [custom services](https://mockoon.com/custom-services/) that allow us to prioritize specific developments based on your needs.
